compose/ui/ui/src/commonMain/kotlin/androidx/compose/ui/semantics/SemanticsProperties.kt

```diff
@@ -449,6 +449,20 @@ private fun <T> throwSemanticsGetNotSupported(): T {
     )
 }
 
+@Suppress("NOTHING_TO_INLINE")
+// inline to break static initialization cycle issue
+internal inline fun <T> AccessibilityKey(name: String) =
```

Comment:
There was a conflict during the last merge here. Please wait `integration` merging for this change

Reply:
oookay. It took some time to rediscover the root of this issue.

I reverted your previous fix after the merge for now. `inline` also works fine, but it requires extra comment with mentioning https://youtrack.jetbrains.com/issue/KT-65040

Reply:
I am upstreaming it in https://android-review.googlesource.com/c/platform/frameworks/support/+/3459419

After merging, I'll revert fork changes and cherry-pick the fix

Reply:
Will do that in a separate PR, with other CL fixes